pages/index.js

```diff
@@ -1,32 +1,124 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/jsx-no-comment-textnodes */
+/* eslint-disable max-len */
 /* eslint react/jsx-filename-extension: 0 */
 /* eslint react/jsx-no-literals: 0 */
 /* eslint arrow-body-style: 0 */
-import React from 'react';
-import Auth from '../server/authentication/Auth';
-
-const auth = new Auth();
+// eslint-disable-next-line react/jsx-no-comment-textnodes
+// eslint-disable-next-line react/jsx-no-undef
+// eslint-disable-next-line jsx-a11y/anchor-is-valid
+// eslint-disable-next-line react/button-has-type
 
-class Callback extends React.Component {
-    componentDidMount() {
-        auth.handleAuthentication();
-    }
-
-    render() {
-        return (
-            <div>Life Aid</div>
-        );
-    }
-}
+import React from 'react';
+import '../scss/home.scss';
 
-const Index = () => {
+const Home = () => {
```

Comment:
go through this component and remove all strings and save them to a `constants.js` file

Reply:
Please through more light on this

Reply:
strings should be defined in your `constants.js` file and imported into the appropriate files that require them

Reply:
You could look at the **File Structure** from https://github.com/Enye-Academy/Engineering-Conventions
This should help. If you still have any questions you can buzz me.

Reply:
Done